kinematics_base/include/moveit/kinematics_base/kinematics_base.h

```diff
@@ -56,6 +56,48 @@ class RobotState;
 namespace kinematics
 {
 
+/*
+ * @enum DiscretizationMethods
+ *
+ * @brief Flags for choosing the type discretization method applied on the redundant joints during an ik query
+ */
+namespace DiscretizationMethods
```

Comment:
why separate namespaces? If we do use them, they should match the ROS style for namespaces.